client/src/styles.scss

```diff
@@ -113,6 +113,7 @@ body {
   text-transform: uppercase;
 }
 
+/////////// Remove flex-layout and replace with CSS ////////////////////
```

Comment:
I'm not sure how long we need this to live on the project. It definitely makes sense now, but it's probably going to only be confusing in a year or two. (Or to students who aren't paying attention to old iteration templates, and this might just be confusing for them.)

Reply:
Good point. I'm not sure either. I wish there was a way to put a giant post-it note on it so we would notice, but it would not show up for students. Since we are still a bit "in process" for replacing flex-layout, I think we should leave it *for now* and plan to remove it. Maybe there should be an issue devoted to removing these types of hints that might be useful only in the short term... and a way to collect what things those are?

Reply:
I think we should leave this in for now.

Reply:
Got it.